access/grpc/client.go

```diff
@@ -28,6 +28,7 @@ package grpc
 import (
 	"context"
 
+	cdcjson "github.com/onflow/cadence/encoding/json"
```

Comment:
Small nit: The format is usually referred to as "JSON-Cadence" or "JSON-CDC" and import aliases are usually `jsoncdc`